android/src/main/java/com/thebluealliance/androidclient/binders/RecyclerViewBinder.java

```diff
@@ -0,0 +1,127 @@
+package com.thebluealliance.androidclient.binders;
+
+import android.support.annotation.Nullable;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.View;
+import android.widget.ProgressBar;
+
+import com.thebluealliance.androidclient.Constants;
+import com.thebluealliance.androidclient.R;
+import com.thebluealliance.androidclient.adapters.ListViewAdapter;
+import com.thebluealliance.androidclient.listitems.ListItem;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import butterknife.Bind;
+import butterknife.ButterKnife;
+import io.nlopez.smartadapters.SmartAdapter;
+import io.nlopez.smartadapters.adapters.RecyclerMultiAdapter;
+
+public class RecyclerViewBinder extends AbstractDataBinder<List<Object>> {
```

Comment:
We no longer need generics. The binding from objects to views is ultimately done by the mapper provided by subclasses of RecyclerViewFragment, which doesn't use generics at all.